PopulateDatabase.py

```diff
@@ -134,11 +138,11 @@ def save_question(question_name=None, question_text=None, choices=None, correct_
     # Members are made to be new by default
     # After the correct number of new members are made, start making expired members
     if number_new < i < (number_new + number_expired):
```

Comment:
If you check active members first, the rest can be expired (improves readability)